releasenotes.md

```diff
@@ -0,0 +1,7 @@
+# 3.0.0.2
+## What's new?
+- Packaged as a NuGet package on https://powershell.myget.org/gallery/powershell-core
+
```

Comment:
The NuGet packages for both OneGet nugetprovider and PackageManagement are stored in https://powershell.myget.org/gallery/powershell-core  feed?

Reply:
c44c680